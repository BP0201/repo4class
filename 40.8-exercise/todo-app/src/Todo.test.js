import { render } from '@testing-library/react';
import Todo from './Todo';

test('renders without crashing', () => {
  render(<Todo />);
});

test("it matches the snapshot", () => {
  const { asFragment } = render(<Todo />)
  expect(asFragment()).toMatchSnapshot();
})