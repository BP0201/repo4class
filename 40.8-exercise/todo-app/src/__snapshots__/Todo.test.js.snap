// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches the snapshot 1`] = `
<DocumentFragment>
  <li>
    <button>
      X
    </button>
  </li>
</DocumentFragment>
`;
